indexer/osm_editor.cpp

```diff
@@ -556,6 +556,21 @@ bool Editor::GetEditedFeatureStreet(FeatureType const & feature, string & outFea
   return true;
 }
 
+vector<uint32_t> Editor::GetFeaturesByStatus(MwmSet::MwmId const & mwmId, FeatureStatus status) const
```

Comment:
It's not an easy task - Editor is a global instance with stored edits file, and is deeply integrated into several libraries.
We probably do some Editor core tests later, before starting refactoring of current implementation.